onnx/defs/reduction/defs.cc

```diff
@@ -239,6 +240,11 @@ The type of the output tensor is integer.)DOC";
         "Whether to select the last index or the first index if the {name} appears in multiple indices, default is False (first index).",
         AttributeProto::INT,
         static_cast<int64_t>(0));
+    schema.Attr(
+        "complement_axis",
+        "Whether to reduce axes complementarily, default is False (reduce target axis). This attribute needs be used simultaneously with axis attribute.",
```

Comment:
Nit: Suggest changing "needs to be used simultaneously" to "is used in conjunction with". This is just to avoid creating the impression that the attribute must be specified if axis is specified.

Reply:
All updated. Thanks.